src/webui.c

```diff
@@ -77,11 +77,37 @@ fetch_wa_cookie(struct evhttp_request *req)
     return wa;
 }
 
+static void
+send_headers(struct evhttp_request *req, const char *contype)
+{
+    struct evkeyvalq *hdrs_out = NULL;
+    hdrs_out = evhttp_request_get_output_headers(req);
+    evhttp_add_header(hdrs_out, "Content-Type", contype);
+}
```

Comment:
Not a fan. Two lines of code moved to function call that hides what it's doing (send_headers? what headers? it's just setting a single content-type header).

Reply:
I did this in case I want to set more / other headers at a later time.

I can revert.

Reply:
There's just no benefit to moving those 2 lines to a function call. Especially as it's a single header.